File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/UnholyPotion.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item;

/* JADX INFO: loaded from: classes3.dex */
public class UnholyPotion extends Item {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.item_unholy_potion_name;
        this.idDescription = R.string.item_unholy_potion_description;
        this.idImage = R.drawable.unholy_potion;
        this.price = 666L;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/UnholyPotion.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item

class UnholyPotion : Item() {
    override fun configureProperties() {
        idName = R.string.item_unholy_potion_name
        idDescription = R.string.item_unholy_potion_description
        idImage = R.drawable.unholy_potion
        price = 666L
    }
}
